caching/src/main/java/com/iluwatar/caching/App.java

```diff
@@ -61,105 +56,157 @@
  */
 @Slf4j
 public class App {
+  /**
+   * Constant parameter name to use mongoDB.
+   */
+  private static final String USE_MONGO_DB = "--mongo";
+  /**
+   * Application manager.
+   */
+  private final AppManager appManager;
+
+  /**
+   * Constructor of current App.
+   *
+   * @param isMongo boolean
+   */
+  public App(final boolean isMongo) {
+    DbManager dbManager = DbManagerFactory.initDb(isMongo);
+    appManager = new AppManager(dbManager);
+    appManager.initDb();
+  }
 
   /**
    * Program entry point.
    *
    * @param args command line args
    */
-  public static void main(String[] args) {
-    AppManager.initDb(false); // VirtualDB (instead of MongoDB) was used in running the JUnit tests
+  public static void main(final String[] args) {
+    // VirtualDB (instead of MongoDB) was used in running the JUnit tests
     // and the App class to avoid Maven compilation errors. Set flag to
     // true to run the tests with MongoDB (provided that MongoDB is
     // installed and socket connection is open).
-    AppManager.initCacheCapacity(3);
-    var app = new App();
+    App app = new App(isDbMongo(args));
```

Comment:
We could add some log output to indicate whether we are using Mongo database engine or something else